### ISSUE TITLE
SonarQube 10 latest image stops on Elastic search

### ISSUE THREAD

**Issue by dib0** [SonarSource/docker-sonarqube#614]
Hi,

I am running SonarQube 10 (latest tag) on Docker on a Synology NAS. Have been running is for over a year. I'm using it in combination with a postgress database.

I have set  sysctl -w vm.max_map_count=262144 on the Linux host. And have set the environment variable SONAR_SEARCH_JAVAADDITIONALOPTS to -Dbootstrap.system_call_filter=false (was an issue with a previous version of the image).

After I updated the image the following error appears:
java.lang.UnsupportedOperationException: seccomp unavailable: CONFIG_SECCOMP not compiled into kernel, CONFIG_SECCOMP and CONFIG_SECCOMP_FILTER are needed

This was previously solved by setting -Dbootstrap.system_call_filter=false, but that doesn't seem to be working correctly? Looking through the documentations and issues I can't find out if anything has changed at this point. Did anything change? Is there an issue with the latest image build?

**Comment by jCOTINEAU** [comment]
Hello @dib0 thanks a lot for taking the time to participate in the community.

Could you tell me what previous version you were using ? 

If it was something like docker 9.9.x+ then nothing changed from the docker part.

Based on your answer i might redirect that topic to the dev team as it might be application related.

Cheers

**Comment by dib0** [comment]
Hi @jCOTINEAU ,

Thank you for replying. I am using the official image with the "latest" tag. I'm not sure about the specific version, but I believe it was version 10. Certainly 9.9.x+

Thanks again!

**Comment by jCOTINEAU** [comment]
Hello again @dib0, 

After some investigation, it looks like the parameter has been removed from elasticsearch 8 ([see here](https://www.elastic.co/guide/en/elasticsearch/reference/master/release-notes-8.0.0.html)).

We upgraded from elastic 7 to 8 inside SonarQube 10.0.

So far we have not found a way to bypass that issue.

I can suggest to use the dedicated sonarqube 9.9 images, which is the supported LTS for a year.

I will inform the dev team about that issue.

Cheers.
